src/status_im2/navigation/roots.cljs

```diff
@@ -2,18 +2,18 @@
   (:require [quo2.foundations.colors :as colors]
             [react-native.platform :as platform]
             [status-im2.navigation.view :as views]
-            [status-im2.navigation.state :as state]
             [status-im2.common.theme.core :as utils.theme]))
 
 (defn status-bar-options
-  []
-  (let [dark-mode? (if (= @state/root-id :shell-stack) (colors/dark?) (utils.theme/dark-mode?))]
-    (if platform/android?
-      {:navigationBar {:backgroundColor colors/neutral-100}
-       :statusBar     {:backgroundColor :transparent
-                       :style           (if dark-mode? :light :dark)
-                       :drawBehind      true}}
-      {:statusBar {:style (if dark-mode? :light :dark)}})))
+  ([dark-root?]
+   (let [dark-mode? (if dark-root? (colors/dark?) (utils.theme/dark-mode?))]
```

Comment:
I'm curious about what was that `:shell-stack`.

I understand we'll pass `true` to this function for the onborading screens we'll create, is it correct?

Reply:
I might need to revert this code. That said, the navigation code was a bit complicated and I did not like coupling the shell-stack to that function when it could be passed as a param. 
I think ideally it could be done that way @ulisesmac but  would appreciate if @Parveshdhull can check it as well.

Reply:
`(colors/dark?)` depends on the user's choice (can only be accessed after login, that's why shell-stack) and `(utils.theme/dark-mode?)` depends on the system theme setting.
So after login we should use the user's choice. And as we didn't had access to that before login, we were using the system theme upto now.

But for new designs, onboarding is always the dark theme, so we can use
`dark-mode? (if (= @state/root-id :shell-stack) (colors/dark?) true)`

Reply:
I reverted this change for now as it's not necessary. I think it would be nice to make this not coupled with the state and just pass it in as a param.